### DISCUSSION TITLE
Python函數return多個值

### DISCUSSION THREAD

**Discussion by ru0372572** [1101-GS4719-C/Python-Community-GS4719-C#792] [General]
作業一第四題需要return很多值，還有換行，文章介紹了return多個值的方法。
[文章連結](https://www.geeksforgeeks.org/g-fact-41-multiple-return-values-in-python/)

PS. 除了文章內的方法，也可以試試將答案加起來(output+='......\n')

**Comment by Pietra1226** [comment]
直接加起來比較快，但很感謝分享，之後可能用得到

**Comment by dinosaur1009** [comment]
對作業真的很有幫助！

**Comment by evanchou1204** [comment]
我也覺得直接加起來比較快，哈哈，但這個可能可以應用在其他更複雜的情況吧。

**Comment by WEI-CHEN-CHUN** [comment]
應該也可以把呼叫class的函式寫在class裡頭
